tests/promptflow/test_promptflow_model_export.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+
+import pytest
+
+from promptflow import load_flow
+from promptflow._sdk.entities._flow import Flow
+
+import mlflow
+from mlflow import MlflowException
+from mlflow.openai.utils import _mock_request, _mock_chat_completion_response, TEST_CONTENT
+
+
+def get_promptflow_example_model():
+
+    flow_path = Path(__file__).parent / "flow_with_additional_includes"
+    return load_flow(flow_path)
+
+
+def test_promptflow_log_and_load_model():
+    model = get_promptflow_example_model()
+    with mlflow.start_run():
+        logged_model = mlflow.promptflow.log_model(model, "promptflow_model")
+
+    loaded_model = mlflow.promptflow.load_model(logged_model.model_uri)
+
+    assert "promptflow" in logged_model.flavors
+    assert str(logged_model.signature.inputs) == "['text': string]"
+    assert str(logged_model.signature.outputs) == "['output': string]"
+
+    assert isinstance(loaded_model, Flow)
+
+
+def test_pyfunc_load_promptflow_model():
+    model = get_promptflow_example_model()
+    with mlflow.start_run():
+        logged_model = mlflow.promptflow.log_model(model, "promptflow_model")
+
+    loaded_model = mlflow.pyfunc.load_model(logged_model.model_uri)
+
+    assert "promptflow" in logged_model.flavors
+    assert type(loaded_model) == mlflow.pyfunc.PyFuncModel
```

Comment:
are there assertions missing here if the test passes with my other comment on the implementation? (path vs object)

Reply:
It doesn't fail because FlowInvoker can accept both str and object type, I've refine the parameter name in `__init__.py`.